zerver/tests/tests.py

```diff
@@ -1552,23 +1552,23 @@ def check_for_toggle_param(self, pattern, param):
 
     # TODO: requires method consolidation, right now, there's no alternative
     # for check_for_toggle_param for PATCH.
-    def check_for_toggle_param_patch(self, pattern, url):
+    def check_for_toggle_param_patch(self, pattern, param):
```

Comment:
`param` made more sense in this case than `url`, and it was also used in the original method.